.travis.yml

```diff
@@ -0,0 +1,32 @@
+---
+language: node_js
+node_js:
+  - "8"
+
+sudo: false
+dist: trusty
+
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+  - name: "npm"
+    before_install:
+      - npm config set spin false
+    install:
+      - npm link
```

Comment:
no `npm install` here?

Reply:
npm link does an npm install at the same time 👍